### PR TITLE
Minor updates to Write partitioning documentation

### PR DESCRIPTION
**Pull request by raunaqmorarka**

### CODE REVIEW

**Review by sopel39 — APPROVED** [review]

**Review by mosabua — APPROVED** [review]

File: docs/src/main/sphinx/admin/properties-write-partitioning.rst [hunk 1]
```diff
@@ -12,15 +12,18 @@ minimum number of partitions, set in ``preferred-write-partitioning-min-number-o
...
 
 ``preferred-write-partitioning-min-number-of-partitions``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 * **Type:** ``integer``
 * **Default value:** ``50``
 
-The minimum number of written partitions that is required to use connector
-``preferred write partitioning``. If the number of partitions cannot be
-estimated from the statistics, then preferred write partitioning is not used.
+Use connector ``preferred write partitioning`` when the optimizer's estimate
```
Comment (m57lyra) [inline docs/src/main/sphinx/admin/properties-write-partitioning.rst:23]: This should not be in backquotes. It is not a property name (use_preferred_write_partitioning is the property name)

**Review by m57lyra — APPROVED** [review]

File: docs/src/main/sphinx/admin/properties-write-partitioning.rst (hunk 1 above)
Comment (raunaqmorarka) [reply docs/src/main/sphinx/admin/properties-write-partitioning.rst]: updated

File: docs/src/main/sphinx/admin/properties-write-partitioning.rst [hunk 2]
```diff
@@ -12,15 +12,18 @@ minimum number of partitions, set in ``preferred-write-partitioning-min-number-o
 are written, each partition is written by a separate writer. As a result, for some connectors such as the
 Hive connector, only a single new file is written per partition, instead of
 multiple files. Partition writer assignments are distributed across worker
-nodes for parallel processing.
+nodes for parallel processing. This can be specified on a per-query basis using
```
Comment (sopel39) [inline docs/src/main/sphinx/admin/properties-write-partitioning.rst:15]: ``use-preferred-write-partitioning`` can be specified on a per...
Similar for the other one

**Review by sopel39 — APPROVED** [review]
small comment

File: docs/src/main/sphinx/admin/properties-write-partitioning.rst (hunk 2 above)
Comment (raunaqmorarka) [reply docs/src/main/sphinx/admin/properties-write-partitioning.rst]: done

**Review by sopel39 — APPROVED** [review]

**Pull request merged**